ajax/libs/country-select-js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "country-select-js",
+  "filename": "js/countrySelect.min.js",
+  "description": "A jQuery plugin for selecting a country",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mrmarkfrench/country-select-js.git"
+  },
+  "keywords": [
+    "country",
+    "international",
+    "jQuery"
+  ],
+  "author": "Mark French",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/mrmarkfrench/country-select-js.git",
```

Comment:
Next time you can use `git` protocol.
Let's bot get new versions via git protocol.
Thanks

Reply:
No mind, it's fine to me.